docker/lite/build.sh

```diff
@@ -14,7 +14,9 @@ fi
 
 # Extract files from vitess/base image
 mkdir base
-sudo docker run -ti --rm -v $PWD/base:/base -u root $base_image bash -c 'cp -R /vt /base/'
+# Ignore permission errors. They occur for directories we do not care e.g. ".git".
+# (Copying them fails because they are owned by root and not $UID and have stricter permissions.)
+docker run -ti --rm -v $PWD/base:/base -u $UID $base_image bash -c '{ cp -R /vt /base/ 2> >(grep -v "Permission denied" >&2); }'
```

Comment:
I actually don't understand it immediately as well :)

I've changed it to the shorter version you suggested and verified that it works.

The initial version was copied from here: http://stackoverflow.com/questions/762348/how-can-i-exclude-all-permission-denied-messages-from-find

Its complex syntax brings features which we don't need e.g. that the return value of `cp` (and not grep) is propagated.